frontend/templates/TabBar.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+const HighlightedTab = styled.h2`
+  border: 0rem;
+  border-bottom: 0.2rem #5f034c;
+  border-style: solid;
+  cursor: pointer;
+  font-size: 1rem;
+  font-weight: bold;
+  margin: 0 1rem;
+  padding: 1rem 0;
+  white-space: nowrap;
+`;
+
+const Row = styled.div`
+  box-shadow: 0rem 0.75rem 1rem -0.5rem lightgrey;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  overflow-y: hidden;
+  wrap: no-wrap;
+`;
+
+const Tab = styled.h2`
+  cursor: pointer;
+  font-size: 1rem;
+  margin: 0 1rem;
+  opacity: 0.2;
+  padding: 1rem 0;
+  white-space: nowrap;
+`;
+
+const TabBar = ({
```

Comment:
Very minor design nit-pick. I noticed that in the design prototype, the selected Tab has the purple underline under it, which extends a little past the actual word in both directions. Attached is a pic to show the difference. The left is the design prototype, the right is the PR. Maybe we can add a little padding to each of the Tabs to achieve this effect
<img width="845" alt="Screen Shot 2021-11-07 at 7 23 10 PM" src="https://user-images.githubusercontent.com/17476012/140667832-82973e07-65ab-4766-b3a4-88f53d440b3d.png">

Reply:
Fixed